Automattic/jetpack
# Update All Sites icon size for Simple Classic

Fixes https://github.com/Automattic/dotcom-forge/issues/7483

## Proposed changes:
Update the `All Sites` icon size on Classic Interface.

| Before | After |
| --- | --- |
| ![sites-icon-before](https://github.com/Automattic/jetpack/assets/402286/5b88b717-148d-4628-8654-8306084bce7b) |![sites-icon-after](https://github.com/Automattic/jetpack/assets/402286/eda2f43d-d2a6-4e45-ac40-079905b6e9cb) |

### Other information:

## Jetpack product discussion
No

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Go to 'wp-admin' with a Classic Interface site (you can switch it in Settings)
* Check the All Sites icon size
